src/gpasswd.c

```diff
@@ -512,23 +494,19 @@ static void log_gpasswd_success (const char *suffix)
 		         "user %s added by %s to group %s%s",
 		         user, myname, group, suffix));
 #ifdef WITH_AUDIT
-		SNPRINTF(buf, "user %s added by %s to group %s%s",
-		         user, myname, group, suffix);
-		audit_logger (AUDIT_USER_ACCT, Prog,
-		              buf,
-		              group, AUDIT_NO_ID,
+		audit_logger_with_group (AUDIT_USER_MGMT, Prog,
+		              "add-user-to-group",
```

Comment:
Maybe we should use enums?  That would centralize an array with all the strings.

```c
enum audit_op {
    AUDIT_ADD_USER_TO_GROUP,
    ...
};

static const char audit_ops[][64] = {
    [.AUDIT_ADD_USER_TO_GROUP] = "add-user-to-group",
    ...
};
```

Reply:
Okay, sounds reasonable.  Anyway, once can easily grep for all such strings, so it's ok.